spec/models/member_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe Member do
+  let(:member) { Member.create!(name: 'John Doe', email: 'jd@artsymail.com', group_ids: [@group.id]) }
```

Comment:
This part is backwards, you're reusing `@group` in a `let`, out of order, instead do:

```ruby
let(:group) { Group.create!(name: 'Design') }
let(:member) { Member.create!(..., groups: [group]) }
```

And you don't need a `before` section.

However there's a bigger issue. Your tests should setup context, and be explicitly verifying something based on that context.

```ruby
describe Member do
  context '#email' do
    it 'stores a lowercase version of e-mail' do 
       # implement this too
    end 
  end 
  context 'with an existing member' do
     let(:member) { }
    context '#email' do
      it 'doesn't allow another member with the same e-mail' do
        # implement
      end
      it 'is case insensitive' do
         expect {
            Member.create!(email: member.email.upcase, ...)
         }.to raise_error ...
      end
    end
  end
end